src/Exceptions/HttpException.php

```diff
@@ -29,11 +30,25 @@ public static function buildHtmlResponse(): HtmlResponse
         $phrase = static::PHRASE;
         $href = static::HREF;
 
+        if (!$code) {
+            $title = $phrase;
+        } else if (strlen($phrase)) {
+            $title = "{$code} - {$phrase}";
+        } else {
+            throw new RuntimeException("Phrase constant is not defined.");
+        }
+
+        if (!strlen($description)) {
```

Comment:
Why don't you have all params that you require for this error in the constructor rather than having them as consts?

Reply:
Easier to generate, but I was thinking of having properties though.

Reply:
For my crappy framework a while ago I did it like this https://github.com/Exanlv/rxak-landviz/blob/master/conf/exceptions.php

Could be something to consider, tho ultimately personal preference I suppose